Libs/Mesh/MeshUtils.cpp

```diff
@@ -168,4 +170,70 @@ int MeshUtils::findReferenceMesh(std::vector<Mesh>& meshes)
   return std::distance(means.begin(), smallest);
 }
 
+Mesh MeshUtils::computeMeanNormals(std::vector<Mesh> &meshes)
```

Comment:
We should be passing a const (see comment below) reference to a vector of std::reference_wrapper(const Mesh) like the bounding box functions in this class.

Reply:
Done